Gemfile

```diff
@@ -1,3 +1,7 @@
 source "https://rubygems.org"
 
+gem "rspec"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.